docs/getting-started/istio/index.md

```diff
@@ -30,32 +30,63 @@ spec:
       stableService: rollouts-demo-stable
       trafficRouting:
         istio:
-          virtualService:
-            # Reference to a VirtualService which the controller updates with canary weights
-            name: rollouts-demo-vsvc
+          virtualServices:
+          # One or more virtualServices can be configured
+          # Reference to a VirtualService which the controller updates with canary weights.
+          - name: rollouts-demo-vsvc1
             routes:
             - primary # optional if there is a single route in VirtualService, required otherwise
+          # Reference to a VirtualService which the controller updates with canary weights
+          - name: rollouts-demo-vsvc2
+            routes:
+            - secondary # optional if there is a single route in VirtualService, required otherwise
```

Comment:
Can we change `secondary` to `primary` like the example in `ocs/getting-started/istio/multipleVirtualsvc.yaml` and put some note saying the route name should match in virtualservice? If not, could you add some explanation the difference between primary and secondary routes?

Reply:
I have added the difference between primary and secondary routes. please check.